test/k8sT/manifests/externalIPs/README.md

```diff
@@ -10,7 +10,7 @@ will run with node-port feature enabled and without kube-proxy.
 ## Create a k8s cluster with 2 nodes *and* NFS
 
 ```bash
-NETNEXT=1 NFS=1 NWORKERS=1 K8S=1 ./contrib/vagrant/start.sh
+KERNEL=net-next NFS=1 NWORKERS=1 K8S=1 ./contrib/vagrant/start.sh
```

Comment:
```suggestion
KERNEL=netnext NFS=1 NWORKERS=1 K8S=1 ./contrib/vagrant/start.sh
```

Reply:
Thanks, the changes were lost when I reordered commits :(.